packages/roosterjs-content-model-types/lib/contentModel/format/ContentModelTableCellFormat.ts

```diff
@@ -12,5 +12,5 @@ export type ContentModelTableCellFormat = ContentModelBlockFormat &
     BorderBoxFormat &
     VerticalAlignFormat &
     WordBreakFormat &
-    TextColorFormat &
+    ContentModelSegmentFormat &
```

Comment:
I prefer not do this since it will allow moving all segment format into cell. We should just keep them in segment level.

When insert new row/column, we can also create new empty paragraph with the segment format.

Reply:
I am just extending the pattern set by TextColorFormat. Previously there was no issue to add segment formatting to the cell.